js-extensions/package.json

```diff
@@ -14,13 +14,14 @@
     "compile": "babel --presets es2015,react src -d dist",
     "test": "gulp test",
     "compile-test": "npm run compile && npm run test",
-    "lint": "gulp lint"
+    "lint": "gulp lint",
+    "prepublish": "npm run compile"
   },
   "author": "Tom Fennelly <tom.fennelly@gmail.com> (https://github.com/tfennelly)",
   "license": "MIT",
   "devDependencies": {
     "@jenkins-cd/eslint-config-jenkins": "0.0.2",
-    "@jenkins-cd/js-builder": "0.0.59",
+    "@jenkins-cd/js-builder": "0.0.60-kz-1",
```

Comment:
you are no german, if you would been this version name is something that you would not use.

Reply:
heh, well it's my initials. I guess I wouldn't have been named this way if I was German ;)